src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] balls = new Lottery[3];
```

Comment:
Назвати масив **Лотерей** ім'ям **Мячі** - це погана ідея. Ти таким чином плутаєш дві різні сутності програми. Нам за завданням не потрыбно мати 3 Лотереї - лише 3 М'ячі.
```suggestion
        Ball[] balls = new Ball[3];
```